backend/source/forms/short-test-form.test.json

```diff
@@ -52,6 +52,10 @@
           ],
           "dependency": null,
           "fn": null,
+          "default_value": {
+            "monitoring": "update",
```

Comment:
```json
"default_value": {
    "submission_type": {
         "monitoring": "update",
         "registration": "new"
    }
}
```

Reply:
Resolved with this commit: 8300fb91b196dfae53dd1313bbd73907356daa2b